app/javascript/lib/metrics-api.js

```diff
@@ -457,6 +458,12 @@ export function log(name, props = {}) {
     props['authenticated'] = true
   }
 
+  if (env === 'development') {
```

Comment:
Is `env === 'development'` different than `scpContext.devMode`?

Reply:
Yes, `devMode` tracks the particular mode the development environment is in, e.g. if it's using Docker.  Logging this will help us assess how much Docker vs. native contributes to local SCP performance.

Reply:
okay cool thanks